testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/sse/SseTest.java

```diff
@@ -86,6 +86,8 @@ public void testAddMessage() throws Exception
             throw new RuntimeException(ex);
          });
          eventSource.open();
+         //Let's wait for SseEventSource to be connected
+         TimeUnit.SECONDS.sleep(3);

```

Comment:
Without this sleep time, invocations of `messageTarget.request().post(Entity.text(msg))` will throw a IllegalStateException on server side since no SseEventSINK is connected yet